File: backend/src/main/java/unb/tppe/domain/entity/Sale.java
package unb.tppe.domain.entity;

import lombok.Getter;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

import java.time.LocalDate;
import java.util.List;

@Getter
@Setter
@SuperBuilder
public class Sale extends BaseEntity {
    private LocalDate dateSale;
    private double price;
    private Client client;
    private Seller seller;
    private List<Product> products;
}

File: front/src/app/entity/Sale.ts
import { BaseEntity } from "./BaseEntity";
import { Client } from "./Client";
import { ProductList } from "./ProductList";
import { Seller } from "./Seller";

export interface Sale extends BaseEntity{
    dateSale: Date;
    price: number;
    products: ProductList[];
    seller: Seller;
    client: Client;
}